docs/specification/2025-03-26/server/tools.mdx

```diff
@@ -221,8 +221,8 @@ Tool results can contain multiple content items of different types:
 
 #### Embedded Resources
 
-[Resources](/specification/2025-03-26/server/resources) **MAY** be embedded, to provide additional context
-or data, behind a URI that can be subscribed to or fetched again by the client later:
+[Resources](resources) **MAY** be embedded to provide additional context
+or data using a suitable [URI scheme](./resources#common-uri-schemes). Servers that use embedded resources **SHOULD** implement the `resources` capability:
```

Comment:
I'm still not sure I would read this and realize that ephemeral resources are allowed. Maybe softening the language to MAY implement resources would help? Or Possibly something like:

Embedded resources MAY be resources already hosted on the server, or MAY be ephemeral resources generated only for this tool response.

Reply:
Agreed, I think explicitly endorsing this use case would be beneficial. Or even citing:

> RFC3986 (7.1 Reliability and Consistency) 
> There is no guarantee that once a URI has been used to retrieve information, the same information will be retrievable by that URI in the future [...]

Without the more explicit guidance, a server developer might feel like they're hacking around the protocol (as I did before this discussion).